### PR TITLE
Create an explicit artifacts directory

### PR DESCRIPTION
**Pull request by deviantintegral**

### CODE REVIEW

File: hooks/RoboFile.php [hunk 1]
```diff
@@ -55,6 +55,17 @@ public function setupSkeleton()
...
+
+        // Create a directory for our artifacts.
+        $this->taskFilesystemStack()
+          ->mkdir('artifacts')
+          ->mkdir('artifacts/phpunit')
+          ->mkdir('artifacts/phpcs')
+          ->run();
+
+        $this->taskFilesystemStack()
+          ->chown('artifacts', 'www-data', TRUE)
```
Comment (juampynr) [inline hooks/RoboFile.php:67]: This is so you don't have to do this at the circle config file?

File: hooks/RoboFile.php [hunk 2]
```diff
@@ -55,6 +55,17 @@ public function setupSkeleton()
         $config->extra->{"enable-patching"} = 'true';
         $config->extra->{"patches"} = new \stdClass();
         file_put_contents('composer.json', json_encode($config));
+
+        // Create a directory for our artifacts.
+        $this->taskFilesystemStack()
```
Comment (juampynr) [inline hooks/RoboFile.php:60]: I think that it is time to add a CHANGELOG so it's easier to figure out what to change when updating the image tag, right?

File: hooks/RoboFile.php (hunk 1 above)
Comment (deviantintegral) [reply hooks/RoboFile.php:67]: Right, and so it's created for every job. I figure even if you aren't using circle having a standard artifacts directory is helpful.

File: hooks/RoboFile.php (hunk 2 above)
Comment (deviantintegral) [reply hooks/RoboFile.php:60]: Yes, my goal is to get to a point where I can tag `0.1.0` and do a changelog. My thinking is that as far as semver goes, we consider any change that requires updating the copied `config.yml` file to be a minor change, then hopefully a major change when we get to 1.0.0.

File: hooks/RoboFile.php (hunk 1 above)
Comment (juampynr) [reply hooks/RoboFile.php:67]: Gotcha. I like that.

**Pull request merged**